framework/web/loader.go

```diff
@@ -168,7 +168,16 @@ func (l *loader) loadActionRoute(basePath, actionName string) string {
 }
 
 func (l *loader) loadActionCallName(basePath, actionName string) string {
-	baseCaller := text.Dot(text.Title(basePath))
+
+	splitPath := strings.Split(text.Title(basePath), " ")
+	for i, controller := range splitPath {
+		// append controller to every controller base name
+		if len(controller) > 0 {
+			splitPath[i] = controller + "Controller"
+		}
+	}
+	baseCaller := strings.Join(splitPath, ".")
```

Comment:
Hmm... this isn't quite the same behavior as what's on the left. IIRC:

```
text.Title("users/posts/comments") => UsersPostsComments
text.Dot("UsersPostsComments") => Users.Posts.Comments
```

I _think_ there's no cases where `splitPath` actually has more than one item:

```
strings.Split(text.Title("users/posts/comments"), " ") => []string{"UsersPostsComments"}
```

I'm kind of surprised the tests would be passing though. Maybe I'm missing something. Can you double-check this and maybe write a test above with a deeply nested controller?